doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -102,11 +102,11 @@ on the cloud.
 Adding or editing tutorials as Jupyter notebooks
 ------------------------------------------------
 
-Under the ``doc/source/notebooks/`` folder of the SciPy tree you can find a few
+Under the ``doc/source/`` folder of the SciPy tree you can find a few
```

Comment:
Does this mean that they can be anywhere within the `doc/source` folder now? Is there anything to add here, or will an `md` file added in this folder "just work"? Where is the Interpolate transition guide now?

Reply:
The .md files will "just work" (be converted as regular html) no matter their location in the doc/source tree. For the "Open as notebook" button to work, .md files will have to be converted to .ipynb files first. Currently, only the ones in the stats folders will be converted but we can add more later. The interpolate transition guide was transformed into a static .md file in #20497

Reply:
I see:
![image](https://github.com/scipy/scipy/assets/6570539/3daf34d3-8458-4a1d-9119-c13625d94932)

OK, thanks!